### PR TITLE
Facets filters

### PR DESCRIPTION
**Pull request by SimonLab**
ready for review
see #156 
@Danwhy @nelsonic some feedback?

### CODE REVIEW

**Comment by nelsonic** [comment]
@SimonLab once you've resolved merge conflicts and added the remaining filters let us know. 😉

File: lib/facets/aggs-all.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ module.exports = function (queryParams) {
       category: {
         filter: {
           bool: {
-            must: filter(queryParams, 'category')
+            must: filter(queryParams)
```
Comment (nelsonic) [inline lib/facets/aggs-all.js]: simpler. :+1:

**Comment by nelsonic** [comment]
@SimonLab code looks good. minor merge conflicts to resolve.
@Danwhy any notes?

**Comment by Danwhy** [comment]
Nope, looks good to me

**Comment by nelsonic** [comment]
Sweet! 🍭

**Pull request merged**